doc/multiple_robot_arms/multiple_robot_arms_tutorial.rst

```diff
@@ -0,0 +1,606 @@
+Introduction
+------------
+In MoveIt, we can plan motions for multiple robot arms, but there are pre-required steps to prepare robot models and configure ROS controllers. This tutorial provides ROS beginners with the steps to model multiple arms, configure controllers, and plan motions using MoveIt.
+
+.. image:: images/multiple_arms_start.png
+   :width: 600pt
+   :align: center
+
+Watch this `Quick YouTube video <https://www.youtube.com/watch?v=h8zlsuzeW3U>`_ of the multiple arms being controlled in Gazebo with the MoveIt Rviz plugin.
+
+Getting Started
+---------------
+If you haven't already done so, make sure you've completed the steps in `Getting Started <https://ros-planning.github.io/moveit_tutorials/doc/getting_started/getting_started.html>`_.
+
+
+The steps of setting multiple arms environments to use MoveIt motion planning are as follows:
+
+1. Build the Xacro/URDF model of the multiple arms.
+
+2. Prepare the MoveIt config package using MoveIt setup Assistant. 
+
+3. Write the ROS controllers configuration and launch files for the multiple arms. 
+
+4. Integrate the simulation in Gazebo with MoveIt motion planning.
+
+5. Plan arms motions with MoveIt Move Group Interface.
+
+This tutorial explains every step to help set up your multiple robot arms environment. 
+
+Step 1: Build the Xacro/URDF model of the multiple arms
+-------------------------------------------------------
+
+The Panda robot arm is used in the following explanation, but the same applies to preparing other types of robot arms.
+
+To start building your multiple arms model, create a new ``panda_multiple_arms`` package as follows: :: 
+
+    cd ~/ws_moveit/src
+    catkin create pkg panda_multiple_arms
+    cd panda_multiple_arms
+    mkdir robot_description && cd robot_description
+    touch panda_multiple_arms.xacro
+
+To prepare your multiple robot arms xacro file (model), you need to have the single arm's xacro file. In the following part, we will build a multiple arms panda robot description file consisting of two identical arms.
+
+
+Our multiple arms model has ``rgt_arm`` and ``lft_arm`` robots. Each arm is equipped with a gripper. Here is a link to the `panda_multiple_arms.xacro <https://github.com/Robotawi/panda_arms_ws/blob/master/src/panda_multiple_arms/robot_description/panda_multiple_arms.xacro>`_ file. Please copy its XML code to your ``panda_multiple_arms.xacro`` file. 
```

Comment:
I would prefer to see `right_arm` instead of `rgt_arm` and `left_arm` instead of `lft_arm`. Hopefully you can do a find/replace in your IDE to change this quickly.

Reply:
Another note for your information, no need to change it:

You can add a planning group that includes both `right_arm` and `left_arm`. I usually call this planning group `both`. Then, you can plan motions for both arms simultaneously, and execute simultaneously.

Reply:
I changed `rgt` to `right` and `lft` to `left`, both in the text and screenshots. Thanks for the suggestion and the beneficial information about the `both` arms planning group!